wireapp/outlook-addin
# Kj/wir 18 reloading wire conversation details

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Once a meeting has been created in Outlook and Wire Meeting details have been added, the user can remove all mention of the link to the meeting in the location field and in the meeting's description.

If it was a mistake and the user wants to revert it, he can't regenerate the meeting invitation link or details by using the "Add Wire meeting Link" button. It will throw an error stating the meeting is already linked.

### Solutions

Reloading Wire conversation details of the already created conversation in a meeting where it has been deleted and setting it again into the appointment.

### Dependencies (Optional)

- [https://github.com/wireapp/outlook-addin/pull/35](https://github.com/wireapp/outlook-addin/pull/35)

Needs releases with:


### Testing

#### How to Test

Create an appointment and click "Add a meeting link." After that, delete the location and/or the body from the appointment. Click again on "Add a meeting link." The deleted part of the appointment should be reloaded.


##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
